contribute.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "Test Pilot",
+    "description": "Test Pilot is a platform for performing controlled tests of new product concepts in Firefox.",
+    "repository": {
+        "url": "https://github.com/mozilla/testpilot",
+        "license": "MPL2",
+        "tests": "https://travis-ci.org/mozilla/testpilot"
+    },
+    "participate": {
+        "home": "https://wiki.mozilla.org/Test_Pilot",
+        "docs": "https://github.com/mozilla/testpilot/blob/master/README.md",
+        "irc": "irc://irc.mozilla.org/#testpilot",
+        "mailing-list": "https://mail.mozilla.org/listinfo/testpilot-dev"
+    },
+    "bugs": {
+        "list": "https://github.com/mozilla/testpilot/issues",
+        "report": "https://github.com/mozilla/testpilot/issues/new"
+    },
+    "keywords": [
+        "css",
+        "django",
+        "docker",
+        "elasticsearch",
```

Comment:
Ok. Removing elasticsearch and pushing an update.

The schema also allows for a `urls` key for dev, prod, or stage... but I wasn't sure if there are any yet?